pkg/provider/handlers/deploy_test.go

```diff
@@ -0,0 +1,47 @@
+package handlers
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/openfaas/faas-provider/types"
+)
+
+func Test_BuildLabels_WithAnnotations(t *testing.T) {
```

Comment:
I would like a few more unit tests if possible?

no annotations + no labels
no annotations but labels
annotations  but no labels

Would it make sense to use a test-table pattern?

https://blog.alexellis.io/golang-writing-unit-tests/